examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -1036,7 +1048,7 @@ def main(args):
                     image.save(image_filename)
 
             del pipeline
-            if torch.cuda.is_available():
+            if accelerator.device.type == "cuda":
```

Comment:
In one of the earlier instances we did `if torch.cuda.is_available():`. Could we maybe stick to that?